apps/client/src/app/components/accounts-table/accounts-table.component.scss

```diff
@@ -4,6 +4,15 @@
   display: block;
 
   .mat-mdc-table {
+    #isExcluded-icon {
```

Comment:
We don't use ids for styling. With my suggestion above, you don't need it at all.